src/methods/create-token.ts

```diff
@@ -75,10 +76,7 @@ export interface CreateTokenInterface<TClientType extends ClientType> {
       : CreateTokenGitHubAppDeviceFlowOptions,
   ): TClientType extends "oauth-app"
     ? Promise<{ authentication: OAuthAppAuth.OAuthAppUserAuthentication }>
-    : Promise<
-        | { authentication: OAuthAppAuth.GitHubAppUserAuthentication }
-        | {
-            authentication: OAuthAppAuth.GitHubAppUserAuthenticationWithExpiration;
-          }
-      >;
```

Comment:
This project is basically yours, "I feel like" sounds like "It's time to learn" to me. But optional token-related keys look good to understand when working with this SDK for the first time and without adding more complexity.

Reply:
@gr2m  I know you have more things to do, but I'm not the most fluent English speaker in the world, and I was having some doubts about the affirmative I said above. Does it sound like "funny," "confusing,"  or "rude" ? I was reviewing my last activities today, I pretended to be a little funny otherwise. :grin: